packages/fxa-content-server/app/tests/spec/models/account.js

```diff
@@ -2972,13 +2972,45 @@ describe('models/account', function() {
           return Promise.resolve(subs);
         });
 
-      account.fetchActiveSubscriptions().then(resp => {
+      return account.fetchActiveSubscriptions().then(resp => {
```

Comment:
This change is not directly related to the issue/PR.  But this test was not running because it was missing the `return`.

Reply:
great catch!